draft-ietf-cdni-uri-signing.xml

```diff
@@ -544,6 +544,33 @@
             claim if no Client IP claim existed in the received
             signed JWT.</t>
         </section>
+        <section anchor="cdnib_claim" title="CDNI Base (cdnib) claim">
+            <t>Base (cdnir) [optional] -
+            The base path for which the token is intended. Its type is a
```

Comment:
So I suggested "base path" as the name, but you said it's more than that. Should this text change then?